src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,56 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+
 public class WorkWithFile {
+    private final String coma = ",";
+    private final String supplyString = "supply";
+    private final String buyString = "buy";
+    private final String resultString = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        try {
+            String finalResult = createReport(readFromFile(fromFileName));
+            writeToFile(toFileName,finalResult);
+        } catch (IOException e) {
+            throw new RuntimeException(e.getMessage());
+        }
+    }
+
+    public String[] readFromFile(String fromFileName) throws IOException {
+        String[] lines = String.join(coma, Files
+        .readAllLines(Paths.get(fromFileName), StandardCharsets.UTF_8)).split(coma);
+        return lines;
```

Comment:
FYI: you can directly return value.
Better logically divide method between lines. Cause in your case `Files` is separated from their method. See the example below of how is better to separate methods and parameters between rows
```suggestion
        return String.join(coma, 
                        Files.readAllLines(Paths.get(fromFileName), StandardCharsets.UTF_8))
                .split(coma);
```